include/fc/io/raw.hpp

```diff
@@ -54,15 +57,15 @@ namespace fc {
     inline void pack( Stream& s, const fc::log_message& msg, uint32_t _max_depth )
     {
        FC_ASSERT( _max_depth > 0 );
-       fc::raw::pack( s, variant(msg), _max_depth - 1 );
+       fc::raw::pack( s, variant(msg), _max_depth - 1 ); // TODO check variant depth?
```

Comment:
FTR - Both TODOs are handled in my variant_fix branch as well, will have to resolve merge conflicts later. Leave them in for now.